examples/smart-mastering-complete/steps/ingestion/persons-ingest-persons.step.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name" : "persons-ingest-persons",

```

Comment:
This looks redundant with ingest-persons - I think this can be deleted, and the flow should point to ingest-persons.